src/recipe/s3-bucket.service.ts

```diff
@@ -51,4 +55,15 @@ export class S3Service {
       throw new ForbiddenException('Could not send image(s) to storage.');
     }
   }
+
+  getPresignedUrlWithClient(key: string) {
```

Comment:
what `Client`? I think this part is not necessary in method name. It can be either `getPresignedUrl` or `getPresignedUrlForKey`

Reply:
In aws docs there were 2 functions - one using client, and one not. Therefore because I'm using client here, I picked this name here. But in this case you're right. I don't need to specify that here.